packages/botonic-react/src/webchat/header.jsx

```diff
@@ -42,15 +42,18 @@ const StyledHeaderTitle = styled(Flex)`
 `
 
 export const DefaultHeader = props => {
+  const { webchatState, useTheme } = props
+  const { theme } = webchatState
   let HeaderImage = Logo
-  if ('brandImage' in props.webchatState.theme)
-    HeaderImage = props.webchatState.theme.brandImage
-  if ('headerImage' in props.webchatState.theme)
-    HeaderImage = props.webchatState.theme.headerImage
+  // These conditions are done in this way because HeaderImage can be null
+  if (getProperty(theme, 'brand.image') || getProperty(theme, 'brandImage'))
```

Comment:
I don't understand how this works when the property is brandImage, and why we support both brandImage and 'brand.image'

Reply:
here we look if one of these two properties is defined in the developer's theme. In case that one of these two is defined we will enter inside this condition. Then two things can happen:
1. The developer has brandImage defined, but the value is `null`. Then the headerImage won't be displayed.
2. The developer has brandImage defined with his image. Then the main headerImage is the one which we define as `brandImage`.